build.xml

```diff
@@ -123,6 +122,19 @@
     </uptodate>
   </target>
 
+  <path id="junit.build.classpath">
+    <fileset dir="${lib.dir}">
+      <include name="junit-4.11.jar"/>
+    </fileset>
+  </path>
+
+  <path id="junit.run.classpath">
+    <path refid="junit.build.classpath"/>
+    <fileset dir="${lib.dir}">
+      <include name="hamcrest-core-1.3.jar"/>
```

Comment:
It turns out that the org.hamcrest classes needed to load the JUnit classes, e.g., Assert, are in junit.jar itself. The hamcrest-core.jar file contains extra classes that might be referenced, but those cannot be needed at runtime unless they were referenced by our code at compile-time.

So we can either just update junit.jar to preserve the status-quo, or include hamcrest-core.jar at compile-time to allow ourselves more freedom.

I'm mildly inclined to use Truth rather than Hamcrest if we want a more extended framework with better error messages, and to stick with just the raw JUnit for now.